Sources/Document/CRDT/RGATreeList.swift

```diff
@@ -333,15 +333,15 @@ class RGATreeList {
     /**
      * `remove` removes the node of the given creation time.
      */
-    func remove(createdAt: TimeTicket, editedAt: TimeTicket) throws -> CRDTElement {
+    func remove(createdAt: TimeTicket, executedAt: TimeTicket) throws -> CRDTElement {
```

Comment:
Since 'editedAt' and 'executedAt' have similar meanings, @7hong13 and I discussed merging them into one. @7hong13 Do you have any opinions?

Reply:
I think merging them into one can be better choice. I was about to use `editedAt` instead of `executedAt`, but both of them seems proper to use as a parameter name.
(I haven't merged them into one yet in Android SDK.)

Reply:
iOS-SDK is unified with executedAt but, It is important to be unified as one, so any modification is possible.

Reply:
then I will unify them with `executedAt` in Android SDK, too.